### DISCUSSION TITLE
Amazing job, this can be combined with auxm to make a Fullstack Framework

### DISCUSSION THREAD

**Discussion by novvoo** [sycamore-rs/sycamore#405] [Ideas]
Amazing job, this can be combined with auxm to make a Fullstack Framework。
MoonZoon is based on Actix ,we prefer  axum to  actix.

**Comment by lukechu10** [comment]
[Perseus](https://github.com/arctic-hen7/perseus) is an existing full-stack framework for Sycamore. It only supports actix-web and warp for now, but I think it would be simpler to add axum support instead of creating something completely new.

**Comment by lukechu10** [comment]
Maybe you want to open an issue there instead?
